### PR TITLE
Add more config option for FreeSWITCH

### PR DESCRIPTION
**Pull request by genofire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ Ansible role for a bigbluebutton installation (following the documentation on ht
 | `bbb_system_locale` | the system locale to use | `en_US.UTF-8` |
 | `bbb_cpuschedule` | CPUSchedulingPolicy | `true` | Disable to fix [FreeSWITCH SETSCHEDULER error][bbb_cpuschedule] |
 | `bbb_freeswitch_ipv6` | Enable IPv6 support in FreeSWITCH | `true` | Disable to fix [FreeSWITCH IPv6 error][bbb_freeswitch_ipv6] |
+| `bbb_freeswitch_external_ip` | Set stun server for sip and rtp on FreeSWITCH | `stun:{{ (bbb_stun_servers | first).server }}` | default is `stun:stun.freeswitch.org |
+| `bbb_dialplan_quality` | Set quality of dailplan for FreeSWITCH | `cdquality` |
+| `bbb_dialplan_energy_level` | Set energy level of dailplan for FreeSWITCH | `100` | only for selected profile `bbb_dialplan_quality`
+| `bbb_dialplan_comfort_noise` | Set comfort noise of dailplan for FreeSWITCH | `true` | only for selected profile `bbb_dialplan_quality`
```
Comment (n0emis) [inline README.md:39]: The default-value for this is 1400. Does changing this to true changes some king of level?

File: tasks/config.yml [hunk 2]
```diff
@@ -224,3 +224,61 @@
   notify:
   - restart freeswitch
   - reload nginx
+
+- name: FreeSWITCH dialplan quality
+  replace:
+    path: /opt/freeswitch/conf/dialplan/default/bbb_conference.xml
+    regexp: '<action application="conference" data="$1@.*"/>'
```
Comment (n0emis) [inline tasks/config.yml:231]: there are two occurences of this line in the file. Do you want to replace just the first or both?

File: README.md [hunk 3]
```diff
@@ -33,6 +33,10 @@ Ansible role for a bigbluebutton installation (following the documentation on ht
 | `bbb_system_locale` | the system locale to use | `en_US.UTF-8` |
 | `bbb_cpuschedule` | CPUSchedulingPolicy | `true` | Disable to fix [FreeSWITCH SETSCHEDULER error][bbb_cpuschedule] |
 | `bbb_freeswitch_ipv6` | Enable IPv6 support in FreeSWITCH | `true` | Disable to fix [FreeSWITCH IPv6 error][bbb_freeswitch_ipv6] |
+| `bbb_freeswitch_external_ip` | Set stun server for sip and rtp on FreeSWITCH | `stun:{{ (bbb_stun_servers | first).server }}` | default is `stun:stun.freeswitch.org |
```
Comment (n0emis) [inline README.md:36]: this line (the "defaukt is ...") is a bit misleading.

File: tasks/config.yml (hunk 2 above)
Comment (n0emis) [reply tasks/config.yml:231]: The regex is currently not working for me. You need to escape the `$` and `/`.

File: README.md (hunk 1 above)
Comment (genofire) [reply README.md:39]: Which is 1400 ? the `bbb_dialplan_energy_level` ? are you sure?

File: tasks/config.yml (hunk 2 above)
Comment (genofire) [reply tasks/config.yml:231]: both

File: README.md (hunk 1 above)
Comment (n0emis) [reply README.md:39]: Yes, the block in the xml-file is the following:
```xml
    <profile name="cdquality">
      <param name="domain" value="$${domain}"/>
      <param name="rate" value="48000"/>
      <param name="interval" value="20"/>
      <param name="energy-level" value="100"/>
      <!-- <param name="sound-prefix" value="$${sounds_dir}/en/us/callie"/> -->
      <param name="muted-sound" value="conference/conf-muted.wav"/>
      <param name="unmuted-sound" value="conference/conf-unmuted.wav"/>
      <param name="alone-sound" value="conference/conf-alone.wav"/>
<!--
      <param name="moh-sound" value="$${hold_music}"/>
      <param name="enter-sound" value="tone_stream://%(200,0,500,600,700)"/>
      <param name="exit-sound" value="tone_stream://%(500,0,300,200,100,50,25)"/>
      <param name="kicked-sound" value="conference/conf-kicked.wav"/>
-->
      <param name="locked-sound" value="conference/conf-locked.wav"/>
      <param name="is-locked-sound" value="conference/conf-is-locked.wav"/>
      <param name="is-unlocked-sound" value="conference/conf-is-unlocked.wav"/>
      <param name="pin-sound" value="conference/conf-pin.wav"/>
      <param name="bad-pin-sound" value="conference/conf-bad-pin.wav"/>
      <param name="caller-id-name" value="$${outbound_caller_name}"/>
      <param name="caller-id-number" value="$${outbound_caller_id}"/>
      <!-- param name="comfort-noise" value="true"/ -->
      <param name="comfort-noise" value="1400"/>

      <!-- <param name="conference-flags" value="video-floor-only|rfc-4579|livearray-sync|auto-3d-position|minimize-video-encoding"/> -->

      <!-- <param name="video-mode" value="mux"/> -->
      <!-- <param name="video-layout-name" value="3x3"/> -->
      <!-- <param name="video-layout-name" value="group:grid"/> -->
      <!-- <param name="video-canvas-size" value="1920x1080"/> -->
      <!-- <param name="video-canvas-bgcolor" value="#333333"/> -->
      <!-- <param name="video-layout-bgcolor" value="#000000"/> -->
      <!-- <param name="video-codec-bandwidth" value="2mb"/> -->
      <!-- <param name="video-fps" value="15"/> -->

    </profile>
```

The one woth the bool is commented out, the other one contains `1400`.

File: tasks/config.yml (hunk 2 above)
Comment (n0emis) [reply tasks/config.yml:231]: yes, both.

File: README.md (hunk 1 above)
Comment (genofire) [reply README.md]: found it on a newer server with it, sorry - here the fix

**Pull request merged**